plugins/mac-capture/mac-window-capture.m

```diff
@@ -102,6 +99,22 @@ static inline void capture_frame(struct window_capture *wc)
 
 	da_init(wc->buffer);
 
+	blog(LOG_INFO, "[window-capture] - Init Display Capture for permissions dialog");
+	
+    struct screen_capture *dc = bzalloc(sizeof(struct screen_capture));
+    if (!dc) {
+        blog(LOG_INFO, "[window-capture] - Display Capture Alloc Fail"); 
+        return NULL;       
+    }
+    dc->display = obs_data_get_int(settings, "display");
+
+    if (!init_screen_stream(dc)) {
+        blog(LOG_INFO, "[window-capture] - Display Capture Init Fail");
+        bfree(dc);
+        return NULL;
+    }
+    bfree(dc);
```

Comment:
wrong indentation for this block

Reply:
appears fine for me

Reply:
could you check again, I still still the indentation issue

Reply:
@florin0x01 ^